COPYRIGHT

```diff
@@ -1,3 +1,6 @@
-Copyright 2008-2014, 2016, 2017, 2019, 2020 Association of Universities for Research in Astronomy, Inc. (AURA)
-Copyright 2015, 2017, 2018, 2019 The Trustees of Princeton University
-Copyright 2015, 2018, 2019 University of Washington
+Copyright 2015-2020, 2022-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+Copyright 2014-2023 The Trustees of Princeton University
+Copyright 2015-2016, 2018-2020, 2022-2023 University of Washington
+Copyright 2015, 2018-2020 The Board of Trustees of the Leland Stanford Junior University, through SLAC National Accelerator Laboratory
+Copyright 2015 The University of Tokyo
+Copyright 2008-2014 LSST Corporation
```

Comment:
All copyrights were transferred from LSST Corp to AURA so this is not the right fix and the original AURA line above was the right one.

Reply:
So just the last line needs to be removed, right?

Reply:
The original AURA dates 2008-2014 were fine and this line can be removed.